alembic/versions/2023_10_30_b6f00cc615cf_sync_db.py

```diff
@@ -0,0 +1,98 @@
+"""Sync db
+
+Revision ID: b6f00cc615cf
+Revises: 392e49db40fc
+Create Date: 2023-10-30 14:57:48.992414
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "b6f00cc615cf"
+down_revision = "392e49db40fc"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        "application", "min_sequencing_depth", existing_type=mysql.INTEGER(), nullable=False
+    )
```

Comment:
I think this will probably fail since there are multiple columns that are null though?

Reply:
Hm, yeah - there are null values in the database. The model should probably be updated to be nullable to reflect this.

Reply:
I think it is out of scope for this PR, I'll create an issue for it and remove it from this revision